ilastik/applets/tracking/annotations/annotationsGui.py

```diff
@@ -1064,15 +1072,15 @@ def _onDivEventPressed(self):
         self._enableButtons(exceptButtons=[self._drawer.divEvent], enable=(not self.divLock))                      
 
 
-    def _setStyleSheet(self, widget, qcolor):                         
+    def _setStyleSheet(self, widget, qcolor, type="QComboBox"):
         values = "{r}, {g}, {b}, {a}".format(r = qcolor.red(),
```

Comment:
Small nit-pick: Here, you're overwriting the meaning of the built-in `type` definition.  I think it's better to avoid that.  (Fun fact: Python 2.7 even allows you to overwrite `False` and `True` if you want -- but don't do that, either! :-)